packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsApprovalFlow/PayableDetailsApprovalFlowSection.tsx

```
@@ -0,0 +1,382 @@
+    const userApprovalCall = approvalCalls.find(
+      (call: any) => call.call === 'ApprovalRequests.request_approval_by_users'
+    );
+
+    const roleApprovalCall = approvalCalls.find(
+      (call: any) => call.call === 'ApprovalRequests.request_approval_by_roles'
+    );
+
+    let assignees: string[] = [];
+    let roleIds: string[] = [];
+
+    if (userApprovalCall && userApprovalCall.params) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)